### PR TITLE
🌐 Add Japanese translation for History Design and Future

### PR DESCRIPTION
**Pull request by komtaki**
This PR translates the  history-design-future.md to Japanese.

According to the issue: #1572 

Thanks!!

### CODE REVIEW

File: docs/ja/docs/history-design-future.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+多くのユースケースに適しているため、既存の方法よりも選ばれています。
+
+多くの開発者やチームが、すでに **FastAPI** にプロジェクトを依存しています（私と私のチームも含めて）。
+
+しかし、まだまだ多くの改善点や機能があります。
+
+**FastAPI**には大きな未来が待っています。
+
+そして、[あなたの貢献](help-fastapi.md){.internal-link target=_blank}に非常に感謝しています。
```
Comment (tokusumi) [inline docs/ja/docs/history-design-future.md:80]: そして、[あなたの助け](help-fastapi.md){.internal-link target=_blank}を大いに歓迎します。

(文脈的に今後の話をしていそうなので、こちらの訳の方があっていそうですが、どうでしょうか)

**Review by tokusumi — CHANGES_REQUESTED** [review]

**Comment by komtaki** [comment]
@tokusumi 

Thanks for the review!! 👍 
I fixed everything. So, check my fix please.

**Review by tokusumi — APPROVED** [review]
LGTM!

File: docs/ja/docs/history-design-future.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## 代替手段
+
+数年前から、私は複雑な要件を持つAPI (機械学習、分散システム、非同期ジョブ、NoSQLデータベースなど) を作成しており、いくつかの開発者チームを率いています。
+
+その一環で、多くの方法を調査し、テストし、利用する必要がありました。
+
+**FastAPI** の歴史は、その前身の歴史が大部分を占めています。
+
+[代替手段とインスピレーション、比較](alternatives.md){.internal-link target=_blank}のセクションでこう述べています:
```
Comment (Attsun1031) [inline docs/ja/docs/history-design-future.md:18]:
```suggestion
[代替ツールから受けたインスピレーションと比較](alternatives.md){.internal-link target=_blank}のセクションでこう述べています:
```

alternatives.mdの日本語訳と整合性を取りました。
https://github.com/tiangolo/fastapi/pull/2043/files

File: docs/ja/docs/history-design-future.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+<blockquote markdown="1">
```
Comment (Attsun1031) [inline docs/ja/docs/history-design-future.md:20]: ここの、alternatives.mdのIntroからの引用についても、整合性をとるとよさそうです。
以下が、現在Openされているalternatives.mdの翻訳PRです。7〜13行目が引用箇所にあたります。

https://github.com/tiangolo/fastapi/pull/2043/files#diff-8c2de13ace5af8415261629c58b6449aR5

**Review by Attsun1031 — COMMENTED** [review]
Thanks for your translation ! Most of the translation felt okay, 

I commented only about the consistency with alternatives.md translation.

**Comment by komtaki** [comment]
@Attsun1031 

Thank you for your comment.
I fixed to match to https://github.com/tiangolo/fastapi/pull/2043/

Please check my fix.

**Review by Attsun1031 — APPROVED** [review]
LGTM👍

**Comment by tokusumi** [comment]
Hi, @tiangolo
This PR has been approved by two reviewers (github actions to label approved PR didn't work, though).
Would you like to require something more for this to be merged?

**Comment by Mause** [comment]
I believe @tiangolo is currently moving houses, so you'll have to be patient

**Comment by tokusumi** [comment]
Oops. I'm really sorry for annoying notification :bow:

**Comment by tiangolo** [comment]
Great! Thank you @komtaki ! :bow: :coffee: 

Thanks for the reviews @Attsun1031 and @tokusumi ! :cake: :nerd_face: 

And thanks @Mause for helping with the clarification! :smile: Indeed, finding a flat in Berlin is quite difficult and has a bunch of complications... I'm still waiting for the internet connection to be installed in my house :sweat_smile: 

But also thanks @tokusumi for pointing out the broken bot :man_facepalming: I wrote one to replace it and it should work fine from now on :robot: :rocket:

**Pull request merged**